PSKoans/Koans/Modules/ActiveDirectory/Introduction/AboutFiltering.Koans.ps1

```diff
@@ -0,0 +1,161 @@
+#Requires -Module ActiveDirectory
```

Comment:
```suggestion
#Requires -Modules ActiveDirectory
```
if I recall my syntax correctly... though I think _technically_ both will work

Reply:
Yep, I make this mistake all the time for this exact reason. I'm just glad it still kind of works even if you do the natural thing instead of the correct thing XD